src/pages/chapter/profile.tsx

```diff
@@ -53,280 +155,313 @@ function ChapterProfilePage() {
-                                newProjects.push(type);
-                              } else {
-                                newProjects = newProjects.filter(
-                                  (t) => t !== type
-                                );
-                              }
-
-                              setChapterProjects(newProjects);
-                            }}
-                          >
-                            {type}
-                          </Checkbox>
-                        ))}
-                      </HStack>
-                    </CheckboxGroup>
-                  </FormControl>
-                </VStack>
-              </Flex>
-            </VStack>
-            <VStack align="start" spacing={5}>
-              <Box>
-                <Text alignSelf="flex-start" fontSize="md" fontWeight={700}>
-                  Project Process Customization
-                </Text>
-                <Text
-                  alignSelf="flex-start"
-                  fontSize="sm"
-                  fontWeight={400}
-                  color="secondaryText"
-                >
-                  Customize your project process for nonprofits to follow.
-                </Text>
-              </Box>
-              <CheckboxGroup value={projectProcess}>
-                <VStack align="start" spacing={2}>
-                  {Object.values(NonprofitStage).map((stage) => {
-                    const isRequired = requiredStages.includes(stage);
-
-                    return (
+                        <Box>
+                          <BsInfoCircle />
+                        </Box>
+                      </Tooltip>
+                    </HStack>
+                  </FormLabel>
+                  <Controller
+                    control={control}
+                    name="maintenancePeriod"
+                    rules={{ required: "Please enter a maintenance period." }}
+                    render={({ field: { ref, onChange, ...restField } }) => (
+                      <NumberInput
+                        {...restField}
+                        max={365}
+                        min={0}
+                        w={36}
+                        onChange={(value) =>
+                          onChange(value ? parseInt(value) : 0)
+                        }
+                      >
+                        <NumberInputField ref={ref} name={restField.name} />
+                        <NumberInputStepper>
+                          <NumberIncrementStepper />
+                          <NumberDecrementStepper />
+                        </NumberInputStepper>
+                      </NumberInput>
+                    )}
+                  />
+                  <FormErrorMessage>
+                    {errors.maintenancePeriod &&
+                      errors.maintenancePeriod.message}
+                  </FormErrorMessage>
+                </FormControl>
+                <FormControl isInvalid={Boolean(errors.maintenanceTypes)}>
```

Comment:
Maintenance types are not pre-populated when first entering the page.  Inputs other than regular text fields can be tricky for this so let me know if you have problems with this.  

This is likely what you need, which is used for the number input above the checkbox group: https://react-hook-form.com/api/usecontroller/controller